contracts/src/CrossChainApplications/GETTING_STARTED.md

```diff
@@ -4,31 +4,70 @@ This section walks through how to build an example cross-chain application on to
 
 ## Step 1: Create Initial Contract
 
-Create a new file called `MyExampleCrossChainMessenger.sol` in the directory that will hold the application.
+Create a new file called `MyExampleCrossChainMessenger.sol` in a `teleporter/contracts/src/CrossChainApplications/MyExampleCrossChainMessenger/` directory.
```

Comment:
Are you clarifying the exact path because of the imports with "../../"?

Reply:
Can you separate and add a line to create a new directory `MyExampleCrossChainMessenger` at the path, then create file under that directory

Reply:
Not sure what you mean by exact path. The remappings don't apply to bash commands. We should circle back and add those remappings to these contracts, but I'm not going to do that in this PR.

Added exact commands as requested.